R/opq.R

```diff
@@ -18,6 +19,9 @@
 #' \dontrun{
 #' overpass_query(q) -> reading_noms
 #' sp::plot(reading_noms$osm_nodes)
+#' p <- overpass_query(opq("Leeds") %>%
+#'   add_feature("leisure", "park"))
+#' sp::plot(p$osm_polygons[1,])
```

Comment:
Hi @mpadge I think there may be some debugging to do here, these things fail for me here (wanted users with mapview to see the parks):

```
#' if(require(leaflet)){
#'   mapview::mapview(p$osm_polygons[1,])
#' }
```

Also, why are the parks visible in the first but not second line below?

```
#' sp::plot(p$osm_polygons[1:10,])
sp::plot(p$osm_polygons)
```

Strange.